.github/release-drafter.yml

```diff
@@ -3,16 +3,17 @@ tag-template: v$NEXT_PATCH_VERSION
 categories:
   - title: 🚀 Features
     labels:
-      - feature
-      - big batch
-      - small batch
+      - 🎥 feature
+      - 🐋 big batch
+      - 🦐 small batch
```

Comment:
I'm not sure if you need the emojis or not here? I think it would use `includes` but not a bad thing to have. Merely just commenting on it.. no need for a change.